go/node/cmd/cli.go

```diff
@@ -100,6 +105,16 @@ func ParseConfigCLI() *NodeConfigCLI {
 	cfg.isDebugNamespaceEnabled = *isDebugNamespaceEnabled
 	cfg.logLevel = *logLevel
 	cfg.isInboundP2PDisabled = *isInboundP2PDisabled
+	cfg.batchInterval = *batchInterval
+	if _, err := time.ParseDuration(cfg.batchInterval); err != nil {
```

Comment:
What if the node validated the time parsing and this mechanism only took strings and simple types ?